src/_locales/ru/messages.json

```diff
@@ -29,23 +29,23 @@
     "html_about_github_title": { "message": "GitHub" },
     "html_about_twitter_title": { "message": "Twitter" },
     "html_about_report_issues_prefix": { "message": "Связаться с нами или сообщить о баге можно на" },
-    "html_about_report_issues_suffix": { "message": ". Пожалуйста. убедитесь что ваша идея или предложение не опубликованы ранее. Спасибо!" },
+    "html_about_report_issues_suffix": { "message": ". Пожалуйста, убедитесь, что Ваша идея или предложение не были опубликованы ранее. Спасибо!" },
     "html_about_debugging_prefix": { "message": "Чтобы включить ведение журнала или установить флаги отладки, перейдите на " },
     "html_about_debugging_suffix": { "message": "страницу отладки." },
     "html_about_the_github_issues_page": { "message": "странице предложений GitHub" },
```

Comment:
I do not see where in this place appeared "space"
![image](https://user-images.githubusercontent.com/23438485/52208478-e04ca880-2891-11e9-8870-c1c26432fee9.png)

Reply:
It's essentially a css issue: https://stackoverflow.com/questions/5078239/how-do-i-remove-the-space-between-inline-block-elements

the fix is to put all the html on a single line. don't worry about it here.

Reply:
OK, I understood